packages/oasis-actions/src/strategies/aave/changeDebt.ts

```diff
@@ -1,21 +1,25 @@
 import { ZERO } from '../../helpers/constants'
-import { AAVEStrategyAddresses } from '../../operations/aave/addresses'
+import { AAVEStrategyAddresses } from '../../operations/aave/v2'
 import { IViewPositionDependencies, IViewPositionParams, WithDebtChange } from '../../types'
 import { AavePosition, AAVETokens } from '../../types/aave'
-import { getCurrentPosition } from './getCurrentPosition'
+import { AaveVersion, getCurrentPosition } from './getCurrentPosition'
 
 export async function changeDebt(
   args: IViewPositionParams<AAVETokens> & WithDebtChange<AAVETokens>,
   { addresses, provider }: IViewPositionDependencies<AAVEStrategyAddresses>,
 ): Promise<AavePosition> {
-  const currentPosition = await getCurrentPosition(args, { addresses, provider })
+  const currentPosition = await getCurrentPosition(args, {
+    addresses,
+    provider,
+    protocolVersion: AaveVersion.v2,
```

Comment:
Make it as constant in parameters. Right now it's a hidden dependency to `aaveV2`. As a client, I don't see it.